### ISSUE TITLE
[Request] Add Chlorophyte Bullet effect to projectiles on command.

### ISSUE THREAD

**Issue by Boii1** [TerraHax/TerraHax-Bin#2]
Basically a command that would allow bullets to seek targets like the Chlorophyte bullet can on command. I've seen this done before, possibly in an older version of this mod or something similar that's outdated.

**Comment by SHACKTINGG** [comment]
Would love to see this one, would be nice to have a feature thats essentially aimbot

**Comment by GooUckd** [comment]
This would like.. Be soo nice.

**Comment by GooUckd** [comment]
But, there's a problem. Chlorophyte only accest enemies, and not players.

**Comment by AlexMarshall123** [comment]
indeed, I'm pretty sure that'd be achievable if you could change the projectile id of bullets though. So a projectile id command would be better.
